viewer/src/datamodels/cad/CadModelUpdateHandler.ts

```diff
@@ -37,18 +38,19 @@ export class CadModelUpdateHandler {
   constructor(sectorRepository: CachedRepository, sectorCuller: SectorCuller) {
     const modelsArray: CadNode[] = [];
     this._updateObservable = combineLatest(
-      this._cameraSubject.pipe(),
+      this._cameraSubject.pipe(auditTime(1000)),
       this._clippingPlaneSubject.pipe(startWith([])),
       this._clipIntersectionSubject.pipe(startWith(false)),
       this._loadingHintsSubject.pipe(startWith({} as CadLoadingHints)),
       this._modelSubject.pipe(
+        share(),
         scan((array, next) => {
           array.push(next);
           return array;
         }, modelsArray)
       )
     ).pipe(
-      auditTime(700),
+      auditTime(250),
```

Comment:
Why?

Reply:
Because you don't want say almost a .7s long delay on things like loading hints / clipping planes etc?

Reply:
I bumped to 0.7s because it's causing lags (probably not on your workstation...). I think we need to have it high until we fix the underlying issue.

Reply:
But if this doesn't affect the camera it might be ok like this as the user doesn't notice the lag like they will with the camera.